app/src/main/java/com/example/mykotlincalculator21/mvp/presenter/CalculatorPresenter.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.mykotlincalculator21.mvp.presenter
+
+import com.example.mykotlincalculator21.mvp.contract.CalculatorContract
+
+class CalculatorPresenter(model: CalculatorContract.CalculatorModelContract, view: CalculatorContract.CalculatorViewContract): CalculatorContract.CalculatorPresenterContract {
+
+    private val model: CalculatorContract.CalculatorModelContract
+    private val view: CalculatorContract.CalculatorViewContract
+
+    init{
```

Comment:
this should be init { (with the space before bracket)

remember Ctrl + Alt + L and Ctrl Alt + O

Reply:
Done. Thx